aips/aip-x-disable-object-burn.md

```diff
@@ -0,0 +1,139 @@
+---
+aip: (this is determined by the AIP Manager, leave it empty when drafting)
+title: Disable Object Burn
+author: gregnazario (greg@aptoslabs.com)
+discussions-to (*optional): <a url pointing to the official discussion thread>
+Status: Draft
+last-call-end-date (*optional): <mm/dd/yyyy the last date to leave feedbacks and reviews>
+type: Standard (Framework)
+created: 8/28/2024
+updated (*optional):
+requires (*optional):
+---
+
+# AIP-X - Disable Safe Burn of User-Owned Objects
+
+## Summary
+
+Safe burn was originally described in [AIP-45](./aip-45.md), which allowed users to tombstone their objects and remove
+direct linkage of an undeletable, also known as *soulbound*, object to an account. This AIP seeks to disable safe
+object burn, as it caused extra complexity, and sometimes unexpected consequences. Owners of burned objects will still
+be able to unburn and recover their tombstoned objects.
+
+### Goals
+
+As a result of this AIP, users will still be able to unburn their burnt objects, but will not be able to burn any new
+undeletable objects. This will reduce complexity and some interesting edge cases associated.
+
+If we delay enacting this, there may be more burnt objects, which could cause more unexpected behaviors around moving
+burnt objects to the burn address.
+
+### Out of Scope
+
+This removes the previous mitigation, since users cannot opt-out of receiving unsolicited content. Wallets or other
+platforms will need to come up with a standardized way of hiding unwanted items.
+
+Additionally, we will not be preventing unburning existing burnt objects at this time.
+
+## Motivation
+
+The purpose here is to remove burn to prevent confusion, and complexity around *soulbound* objects changing owners.
+Originally, this was meant mostly for being able to hide unwanted *soulbound* NFTs. But, there are more use cases for
+*soulbound* objects (e.g. primary fungible store), which can complicate functionality around them. Removing this
+functionality simplifies cases for builders, and can make it easier to accomplish things like allowlists on fungible
+assets.
+
+If we do not accept this proposal, some functionality around *soulbound* objects will require extra complexity or even
+may not be able to work around burns.
+
+## Impact
+
+This impacts the following parties:
+
+* Users who have already burnt, or plan to burn owned *soulbound* objects
```

Comment:
can we show some data on how many burns happened in the past to understand the impact better?

Reply:
I do not have this data as it is not indexed.

The complexity of handling indexing for something that was purely driven because a couple engineers didn't like having soulbound NFTs in their accounts is low.

It's not handled by any exchange, or by any NFT tooling, and is barely documented.

Reply:
how does it impact those that have already burnt?

Reply:
It does not impact already burnt objects.  However, objects that are burnt can still be unburnt.

Let me add that.